bindings/python/verovio.i

```diff
@@ -1,6 +1,6 @@
 /////////////////////////////////////////////////////////////////////////////
 // Name:        verovio.i
-// Author:      Laurent Pugin
+// Author:      Laurent Pugin and Klaus Rettinghaus
```

Comment:
The author of a file is creator of it. We are not going to start adding the names of everybody making changes to the files in the repository.